libvirt/tests/src/virsh_cmd/host/virsh_maxvcpus.py

```
@@ -25,6 +25,11 @@ def run(test, params, env):
     (3) Call virsh maxvcpus with an unexpected option
     """
 
+    # Checking libvirt version before running maxvcpus with kqemu
+    if libvirt_version.version_compare(2, 19, 2):
+        if "kqemu" in option:
```

Ruff (0.14.10)

Undefined name `option`

(F821)